### PR TITLE
long term consistency checker

### PR DESCRIPTION
**Pull request by g-harel**

### CODE REVIEW

File: consistency/test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    time: "time",
+    query: "query",
+    hash
+});
+
+it("should return a hash if it is not in the item", () => {
+    expect(typeof check(item(undefined))).toBe("string");
+});
+
+it("should return false when the hash does not match", () => {
```
Comment (shelbaz) [inline consistency/test.js:14]: is this a valid test for not matching hash? Isnt this more like a null hash? Even empty string generates "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" on SHA-256

Comment (g-harel) [reply consistency/test.js:14]: the thing being hashed is `id + time + query`

this test is just checking that the behavior is correct when the hash string (stored in the item) is different than the one computed from the values in the item.

Comment (shelbaz) [reply consistency/test.js:14]: ok fair

**Review by shelbaz — APPROVED** [review]
All good

**Comment by emd21** [comment]
Looks good!

**Pull request merged**